aspnetcore/security/authentication/microsoft-logins.md

```diff
@@ -0,0 +1,125 @@
+---
```

Comment:
See comments on the other docs for style issues common to all of them -- log in vs sign in, section vs page or article, no caps in file names, references to Azure in Next Steps, etc.